mesonbuild/scripts/pycompile.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2016 The Meson development team
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# ignore all lints for this file, since it is run by python2 as well
+
+# type: ignore
+# pylint: disable=deprecated-module
+
+import json, os, subprocess, sys
+from compileall import compile_file
+
+quiet = int(os.environ.get('MESON_INSTALL_QUIET', 0))
+
+def compileall(files):
+    for f in files:
+        # length of {py_xxxxlib} = 12
```

Comment:
I did not understand that immadiately, maybe expand a bit the comment saying that f starts with `{py_purelib}` or `{py_platlib}` which are both 12 char by chance.

Reply:
Fixed.